spec/requests/users/registrations_spec.rb

```diff
@@ -1,15 +1,15 @@
 require 'rails_helper'
 
 RSpec.describe "Users::Registrations", type: :request do
-  describe "POST /users" do
-    it "should not create an account for weak passwords" do
-      post "/users", params: { user: { email: "test@example.com", password: "12345678" } }
-      expect(response).to have_http_status(:bad_request)
-    end
+  # describe "POST /users" do
+  #   it "should not create an account for weak passwords" do
+  #     post "/users", params: { user: { email: "test@example.com", password: "12345678" } }
+  #     expect(response).to have_http_status(:bad_request)
+  #   end
```

Comment:
Since this is presumably temporary, we'd generally just mark them as "temporarily skipped" like so with `xit`:

```suggestion
  describe "POST /users" do
    xit "should not create an account for weak passwords" do
      post "/users", params: { user: { email: "test@example.com", password: "12345678" } }
      expect(response).to have_http_status(:bad_request)
    end
```

This way, you'll be reminded that there are still tests to fix whenever you read the output of your test run.